### DISCUSSION TITLE
Leantime V2.2.2

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#810] [Announcements]
## Highlights
* Leantime has received a facelift. Lighter theme and CSS cleanup
* To-Dos now open in a modal making it a lot quicker to manage tasks from any screen

## What's Changed
* small update to release package creator by @marcelfolaron in https://github.com/Leantime/leantime/pull/790
* Create package-lock.json by @marcelfolaron in https://github.com/Leantime/leantime/pull/791
* Change filter_var to htmlspecialchars due to deprecation of FILTER_SANITIZE_STRING by @funkywaddle in https://github.com/Leantime/leantime/pull/793
* Fix spacing issue of newly added ToDo status by @funkywaddle in https://github.com/Leantime/leantime/pull/799
* Use config object already instantiated by @funkywaddle in https://github.com/Leantime/leantime/pull/802
* updating method docstring to reflect correct scope by @funkywaddle in https://github.com/Leantime/leantime/pull/803
* Facelift by @marcelfolaron in https://github.com/Leantime/leantime/pull/805
* Facelift - ticket url open in modal and misc clean up by @marcelfolaron in https://github.com/Leantime/leantime/pull/806
* Facelift by @marcelfolaron in https://github.com/Leantime/leantime/pull/807
* Ensure to include hidden files and small updates to css by @marcelfolaron in https://github.com/Leantime/leantime/pull/808

## New Contributors
* @funkywaddle made their first contribution in https://github.com/Leantime/leantime/pull/793

**Full Changelog**: https://github.com/Leantime/leantime/compare/v2.2.1...v2.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Leantime/leantime/releases/tag/v2.2.2'>Leantime V2.2.2</a>.</em>

**Comment by VINISHVV** [comment]
It seems the Lean time alignment for todo's and other pages seems broken.

Also is there any option to change the left side background clour from"white" to any other colour.

**Reply by marcelfolaron** [reply]
Hi, I just released 2.2.3 last night which solves some of the design issues you mentioned. 
Please update and let me know if that solves the issue

**Reply by VINISHVV** [reply]
@marcelfolaron - Thanks for quick response.
I am currently using the latest version 2.2.3 but still showing some design issues.

**Comment by marcelfolaron** [comment]
Can you show me a screenshot of the issues?

**Reply by VINISHVV** [reply]
![image](https://user-images.githubusercontent.com/65057910/181251546-cb1648ee-bd03-4472-a4a1-5dc93bf3995f.png)
Please find the above image.

**Comment by marcelfolaron** [comment]
I see. Regarding the left bar:
You can change the background color by adjusting the left bar colors in
public/css/themes/Leantime-default.css

(The next release will allow you to customize the entire theme)

For the 2nd issue: I’ll have to check what’s going on there.

The kanban board being scrollable is most likely monitor size. We set a min
width for kanban columns and it seems that in this case 5 columns are
larger than the monitor size allows. You can remove/hide columns under
project settings.

Let me check in a couple things and get back to you.

Sincerely,
Marcel

**Reply by VINISHVV** [reply]
@marcelfolaron - Thanks, If i hide the left menu then its working fine. 
I feel the alignment needs to correct in every case.

![image](https://user-images.githubusercontent.com/65057910/181257817-ff049d5d-6457-408a-ac50-cde6650eac6a.png)
